### PR TITLE
Dev/admin role

### PR DESCRIPTION
**Pull request by bakunyo**
# Roleの有効化

これまで全ユーザーが管理画面にログインできるようになっていたので、admin権限をもったユーザーのみに限定した

### CODE REVIEW

File: spec/requests/admin/dashboard_spec.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+describe Admin::DashboardController, type: :request do
+  let(:user) { create :user }
+  let!(:role) { create :user_role, user: user, admin: admin_flag }
+  before { login user }
+
+  describe "can login to active_admin only admin role" do
```
Comment (freezer530000) [inline spec/requests/admin/dashboard_spec.rb]: **[rubocop]**
- Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)

File: spec/requests/admin/dashboard_spec.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    before { get admin_root_path }
+
+    context "admin user" do
```
Comment (freezer530000) [inline spec/requests/admin/dashboard_spec.rb]: **[rubocop]**
- Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)

File: spec/requests/admin/dashboard_spec.rb [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+      let(:admin_flag) { true }
+      it { expect(response).to have_http_status :success }
+      it { expect(response.body).to match 'ダッシュボード' }
+    end
+
+    context "not admin user" do
```
Comment (freezer530000) [inline spec/requests/admin/dashboard_spec.rb]: **[rubocop]**
- Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)

**Comment by ta-chibana** [comment]
LGTM

**Comment by Pujyoooo** [comment]
LGTM

**Comment by Pujyoooo** [comment]
マージします

**Pull request merged**